warehouse/query-core/src/main/java/datawave/query/jexl/nodes/ExceededOrThresholdMarkerJexlNode.java

```diff
@@ -34,6 +34,12 @@
     public static final String EXCEEDED_OR_FIELD = "field";
     public static final String EXCEEDED_OR_PARAMS = "params";
     
+    private static final String LABEL = "EOTM";
```

Comment:
Consider shortening this to EOT, or even EO since the 'threshold' and 'marker' parts are sort of implied.

Reply:
Making this change wouldn't be a problem, but first, did you see @ivakegg's comment above about using shorter labels that aren't as short as acronyms? Any thoughts on that?

Reply:
@jwomeara We need your feedback relative to coming up with a compromise between acronyms and something a little more understandable by the non-query savvy bystander .

Reply:
I'm all about them acronyms.  *shrug*

What would the middle-ground look like?  Can you give me an example @ivakegg ?

Reply:
@jwomeara, @ivakegg provided the follow examples above:

> I like shorter labels, but I feel those are too short. How about:
List (for exceededOrThreshold), Bounded, Delay, Hole, Eval, Term, and Value.

Reply:
@jwomeara pinged me on the side and is OK with whatever labels we agree to.  So lets use the short word labels instead.  Start with the ones I suggested.  Thanks.

Reply:
@lbschanno Important safety tip!  I just remembered that these labels actually translate to fields that are added into the JexlContext during the evaluation of the query.  So please add an underscore at the beginning and end of each of these labels.  Thanks.

Reply:
@ivakegg good advice, will do.

Reply:
@ivakegg I changed the labels.